benchmark/db/schema.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+ActiveRecord::Schema.define(version: 0) do
```

Comment:
It looks like this was copied from `spec/db/schema.rb`. If we think benchmarks and correctness tests will use the same schemas can we share that file across both types of tests. Alternatively we think the schemas for benchmarks and correctness tests will diverge, can we slim down `benchmark/db/schema.rb` to just the schema elements needed by the current benchmarks.